src/main/resources/dbmigration/V6__STUDENT.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO STUDENT(ID, NAME);
```

Comment:
I would create a separate PR for the SQL files only and then the second one for the controllers/repositories/business logic.

Reply:
more than that, by our UML diagram we won't have such tables at all.